docs/source/plotly_figures/plot_contour.html

```diff
@@ -0,0 +1,24 @@
+<html>
+<head><meta charset="utf-8" /></head>
+<body>
+    <div>
+        
+        
+            <div id="c7a563a1-e56b-4419-93c4-3354a268588a" class="plotly-graph-div" style="height:100%; width:100%;"></div>
```

Comment:
`id` here is generated by `uuid.uuid4()`. Plotly doesn't provide an option to make this deterministic.

https://github.com/plotly/plotly.py/blob/65625d49226632abcf932d530aa9c90ba1bc59c1/packages/python/plotly/plotly/io/_html.py#L134

Reply:
We could define the function below and replace the id value with some deterministic value. 

```python
import re

def replace_id(html, new_id):
    old_id = re.search(r'<div id="([0-9a-z-]+)"', html).group(1)
    return html.replace(old_id, new_id)
```

Reply:
Ah good catch. I think we can allow this kind of diff though. What do you think?

Reply:
Agree on allowing this diff.